testplan/testing/multitest/base.py

```diff
@@ -606,6 +606,7 @@ def _run_case_related(method):
 
         # native assertion objects -> dict form
         testcase_report.extend(case_result.serialized_entries)
+        testcase_report.attachments.extend(case_result.attachments)
```

Comment:
I'm not too sure what this is doing, is this converting from multi test attachment -> TestCaseReport attachments?

Reply:
In general, this section of code is converting from a `Result` object into a `Report` object, they are subtly different. Here we are copying the list of attachments from the result object onto the report object.